packages/node_modules/@webex/internal-plugin-encryption/src/kms.js

```diff
@@ -436,6 +437,13 @@ const KMS = WebexPlugin.extend({
           return this.request(payload, {onBehalfOf});
         }
 
+        // KMS Error. Notify the user
+        if (reason.kmsError) {
```

Comment:
Let's use the `KMSError` here
```suggestion
        if (reason instanceof KMSError) {
```